stackrc

```diff
@@ -138,13 +138,15 @@ if [[ -r $RC_DIR/.localrc.password ]]; then
 fi
 
 # Control whether Python 3 should be used at all.
-export USE_PYTHON3=$(trueorfalse False USE_PYTHON3)
+#export USE_PYTHON3=$(trueorfalse False USE_PYTHON3)
+export USE_PYTHON3=False
```

Comment:
This should be set in local.conf, not changed here

Reply:
I will move it.